media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -958,6 +971,16 @@ textarea:focus {
     }
 }
 
+.inquiry-choices {
+    ul {
+        list-style: none;
+    }
+
+    .option-double {
+        width: 440px;
```

Comment:
This is too wide in small viewports (320px is the narrowest we really plan for). You can add another rule in the mobile media query setting this to <code>width: auto;</code>